.golangci.yml

```diff
@@ -0,0 +1,200 @@
+run:
```

Comment:
What is the usage of this file?

Reply:
it's required for local linting.  
[golangci-lint](https://github.com/golangci/golangci-lint)